OpenEdXMobile/src/main/java/org/edx/mobile/view/custom/URLInterceptorWebViewClient.java

```diff
@@ -179,6 +180,12 @@ public boolean shouldOverrideUrlLoading(WebView view, String url) {
         }
     }
 
+    @TargetApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
Why are we setting the Target API for this function, especially setting target API to LOLLIPOP?

Reply:
We know that this function won't be called prior to nougat (API level 24) and in its body, we are using a function that was introduced in lollipop (API level 21), hence the annotation.
<img width="759" alt="Screenshot 2019-10-18 at 3 08 04 PM" src="https://user-images.githubusercontent.com/1710804/67085882-1bd99980-f1b9-11e9-8a90-71d19ecaf6ec.png">